charts/metallb/templates/speaker.yaml

```diff
@@ -147,6 +147,15 @@ spec:
       terminationGracePeriodSeconds: 0
       hostNetwork: true
       volumes:
+      {{- if .Values.speaker.memberlist.enabled }}
+        - name: memberlist
+          secret:
+            secretName: {{ include "metallb.secretName" . }}
+            defaultMode: 420
+            items:
+            - key: secretkey
+              path: metallb/speaker
```

Comment:
just "key" as the rest is given by the full path

Reply:
done